src/Microsoft.NetCore.Analyzers/UnitTests/Security/DoNotDisableHTTPHeaderCheckingTests.cs

```diff
@@ -47,7 +47,27 @@ public void TestMethod()
         httpRuntimeSection.EnableHeaderChecking = flag;
     }
 }",
-            GetCSharpResultAt(11, 9, DoNotDisableHeaderChecking.Rule));
+            GetCSharpResultAt(11, 9, DoNotDisableHTTPHeaderChecking.Rule));
+        }
+
+        [Fact]
+        public void TestPropertyInitializerDiagnostic()
+        {
+            VerifyCSharp(@"
+using System;
+using System.Web.Configuration;
+
+class TestClass
+{
+    public void TestMethod()
+    {
+        var httpRuntimeSection = new HttpRuntimeSection
+        {
+            EnableHeaderChecking = false
```

Comment:
Ah, this generates an `ISimpleAssignmentOperation`, sorry I thought otherwise. Thanks for adding the unit tests.